### DISCUSSION TITLE
Another issue with _document.jsx

### DISCUSSION THREAD

**Discussion by KronsyC-old** [vercel/next.js#29079] [Help]
### What version of Next.js are you using?

11.1.2

### What version of Node.js are you using?

16.6.2

### What browser are you using?

Chrome

### What operating system are you using?

Windows

### How are you deploying your application?

Vercel

### Describe the Bug

I Ported my `pages` directory to `/src/pages`, but i soon figured out that having a `_document.jsx` in this directory throws an error

### Expected Behavior

The Application to build without issue

### To Reproduce

1. Place your _document.jsx inside /src/pages/_document.jsx

2. Attempt to build your application

3. Error

**Comment by housseindjirdeh** [comment]
Hi! Could you share the error message you're seeing? :)
